packages/typespec-ts/eng/scripts/copy-static-helpers.js

```diff
@@ -0,0 +1,43 @@
+import { copyFile, mkdir, readdir } from "fs/promises";
+import path from "path";
+import { fileURLToPath } from "url";
+
+// Get the correct __dirname in both Windows and Linux
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+async function copyFiles(srcDir, destDir, fileFilter) {
+  try {
+    const entries = await readdir(srcDir, { withFileTypes: true });
+    await mkdir(destDir, { recursive: true });
+
+    for (const entry of entries) {
+      const srcPath = path.join(srcDir, entry.name);
+      const destPath = path.join(destDir, entry.name);
+
+      if (entry.isDirectory()) {
+        await copyFiles(srcPath, destPath, fileFilter);
+      } else if (entry.isFile() && fileFilter(srcPath)) {
+        await copyFile(srcPath, destPath);
+      }
+    }
+  } catch (error) {
+    console.error(`Error copying files from ${srcDir} to ${destDir}:`, error);
+    process.exit(1); // Exit with a non-zero status code on error
+  }
+}
+
+const srcDir = path.resolve(__dirname, "../../src/modular/static-helpers");
+const destDir = path.resolve(
+  __dirname,
+  "../../dist/src/modular/static-helpers"
```

Comment:
Yes, I'll move this to static at the root of the project. As for including it in files I will have to test a bit more since resolving these paths is not super straight forward with ESModules, so not sure if they being outside of ./dist would cause problems. I'll test though

Reply:
It seems like it works, removed the copy script